SPECS/SymCrypt-OpenSSL/SymCrypt-OpenSSL.spec

```diff
@@ -76,7 +68,21 @@ fi
 %{_includedir}/e_scossl.h
 %{_sysconfdir}/pki/tls/symcrypt_prov.cnf
 
+# The log directory for certsinuse logging has permissions set to 1733.
+# These permissions are a result of a security review to mitigate potential risks:
+# - Group and others are denied read access to prevent user-level code from inferring
```

Comment:
Just curious about the group permissions: why prevent "root" group read access to the folder? I'm wondering, if this wouldn't prevent legitimate actions from being executed.